pkg/keystore/mem/service.go

```diff
@@ -15,11 +15,17 @@ import (
 
 var _ keystore.Service = (*Service)(nil)
 
+// Service is the memory-based keystore.Service implementation.
+//
+// Keys are stored in the in-memory map, where the key is the name of the
+// private key, and value is structure where actual private key and password are
+// stored.
```

Comment:
// Keys are stored in an in-memory map, where the key is the name of the
// private key, and the value is the structure where the actual private key and the password are
// stored.